starter_code/src/movies.js

```diff
@@ -1,20 +1,190 @@
 /* eslint no-restricted-globals: 'off' */
+
+
+// ************ turnHoursToMinutes hacer con un reduce *****************/
 // Turn duration of the movies from hours to minutes 
 
+function turnHoursToMinutes(movieArray) {
+
+    return movieArray.map(function (item) {
+        var newMovieArry = Object.assign({}, item);
+        var transformToMinutes = newMovieArry.duration.split(' ');
+
+        if (transformToMinutes.length === 2) {
+            var hour = Number(transformToMinutes[0].slice(0, transformToMinutes.indexOf('h'))) * 60;
+            var minutes = Number(transformToMinutes[1].replace('min', ''));
+            newMovieArry.duration = hour + minutes;
+        }
+
+        if (transformToMinutes.length === 1) {
+            if (transformToMinutes[0].includes('h')) {
+                newMovieArry.duration = Number(transformToMinutes[0].slice(0, transformToMinutes.indexOf('h'))) * 60;
+            }
+            else if (transformToMinutes[0].includes('min')) {
+                newMovieArry.duration = Number(transformToMinutes[0].replace('min', ''));
+            }
+
+        }
+        return newMovieArry
+    })
```

Comment:
Demasiado largo y demasiados condicionales, podéis acortarlo? Además sería muy aconsejable que uséis varias funciones más cortas y que os puedan ayudar a reutilizar código más adelante